lib/buidlerProvider.ts

```diff
@@ -0,0 +1,30 @@
+import {createProvider} from '@nomiclabs/buidler/internal/core/providers/construction';
+import {EthersProviderWrapper} from '@nomiclabs/buidler-ethers/dist/ethers-provider-wrapper';
+import {BuidlerNetworkConfig} from '@nomiclabs/buidler/types';
+import defaultAccounts from './config/defaultAccounts';
+
+const defaultBuidlerOptions = {
+  accounts: defaultAccounts.map((account) => ({
+    balance: account.balance,
+    privateKey: account.secretKey
+  })),
+  blockGasLimit: 8000000000000,
+  hardfork: 'istanbul',
+  throwOnTransactionFailures: true,
+  throwOnCallFailures: true,
+  chainId: 17
+  };
```

Comment:
Any opinions on the defaults? Especially `blockGasLimit`, `hardfork` and `chainId `.